src/manage_nsfs/manage_nsfs_cli_utils.js

```diff
@@ -90,8 +90,18 @@ function get_boolean_or_string_value(value) {
     }
 }
 
+function parse_comma_delimited_string(value) {
+    if (typeof value === 'object') {
+        return value;
+    }
+    if (typeof value === 'number') {
+        return [value];
+    }
+    return value.split(',').map(val => Number(val));
```

Comment:
considering we by this point we validate that this value can only be string of the correct format (comma delimited list of positive numbers). it can't fail. even if its a single number it will be correct. if by any chance we get a value that doesn't fit the format this function will return 'nan'

Reply:
I changed it to also run split only if its a string type. for all other types not covered by this function, including undefined I change it to just return the value